homeassistant/components/tessie/strings.json

```diff
@@ -117,6 +117,14 @@
         "name": "Passenger temperature setting"
       }
     },
+    "cover": {
+      "windows": {
+        "name": "Vent windows"
```

Comment:
Vent?

Reply:
So it doesnt actually open the windows fully, it only opens them a little bit to "vent" the air. The API call is literally `vent-windows`. https://developer.tessie.com/reference/vent-windows

Reply:
Aaah